.github/workflows/hashes.yaml

```diff
@@ -17,13 +19,43 @@ jobs:
       - uses: actions/checkout@v2
       - name: Install dependencies
         run: |
-          sudo apt-get install -y --no-install-recommends curl gpg jq python3-pip
+          sudo apt-get install -y --no-install-recommends curl gpg jq python3-pip zbar-tools
           sudo pip3 install yq
-      - name: Verify hashes.txt signature
+      - name: Verify hashes.txt + contributing.yml signature
         run: |
           curl -sL https://raw.githubusercontent.com/monero-project/monero/master/utils/gpg_keys/binaryfate.asc |
             gpg --import
           gpg --verify downloads/hashes.txt
+          # signed by myself for test purposes only.
+          curl -sL https://raw.githubusercontent.com/plowsof/pgp/refs/heads/main/plowsofsmol.asc |
+            gpg --import
+          gpg --verify _data/contributing.yml.asc
+      - name: Verify General Fund donation QR's
+        run: |
+          yaml="_data/contributing.yml"
+          get_yaml_value() {
+              awk -v key="$1:" '$1 == key {print $2}' "$yaml"
```

Comment:
I'm not familiar with awk but looks like easy to break things when the content of the site changed a bit. Is it?

Reply:
awk is a text processor and $1 processes the first argument $2 the second etc.

an example is
`echo uhh nah | awk '{print $2 $1}'

Reply:
also, we're only looking at "contributing.yml" . this is a hack to access yaml variables. astro has no problem importing yml files / using their variables so the same signed file will be usable by both.